client/sites-dashboard/components/sites-search.ts

```diff
@@ -0,0 +1,27 @@
+import Search from '@automattic/search';
+import styled from '@emotion/styled';
+
+export const SitesSearch = styled( Search )`
+	--color-surface: #f6f7f7;
+	border-radius: 4px;
+	overflow: hidden;
+	height: 44px;
+
+	.search-component__icon-search {
```

Comment:
This should not be necessary. Ideally, we could pass to `Search` either:
- a `searchIconClassName` prop, allowing to customize the existing icon;
- a custom `searchIcon` prop, allowing to pass a custom icon.